### PR TITLE
Add auto refresh rate configuration into Dashboard

### LINKED ISSUE #13: Add auto refresh rate configuration
It would be nice to have a dropdown or something similar to let the client decide the refresh rate of the dashboard.

### PR DESCRIPTION
**Pull request by LucianoHanna**
## Closes #13

### CODE REVIEW

**Comment by LucianoHanna** [comment]
I add the refresh rate to a navbar and highlight which is active

**Comment by LucianoHanna** [comment]
I refactored my code to use material-ui

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@material-ui/core": "^4.4.0",
     "@material-ui/icons": "^4.4.3",
+    "bootstrap": "^4.3.1",
```
Comment (marcelometal) [inline package.json]: please, remove it ;)

File: src/index.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (marcelometal) [inline src/index.js]: please, remove it ;)

**Comment by rafaveira3** [comment]
Hello, @LucianoHanna! We have no words to thank you for your time and effort in working on this issue. Unfortunately, we internally discussed this feature and decided that we may redesign how we display our huskyCI results in our dashboard. We have closed this issue for now!

Your PR was very good, by the way! 😊

**Pull request closed without merging**